src/styles/styles.js

```diff
@@ -760,7 +760,7 @@ const styles = {
     baseTextInput: {
         fontFamily: fontFamily.EXP_NEUE,
         fontSize: variables.fontSizeNormal,
-        lineHeight: variables.lineHeightNormal,
+        lineHeight: variables.lineHeightXLarge,
```

Comment:
Here's reason: https://github.com/Expensify/App/issues/14674#issuecomment-1419053266
As you see, emoji on 1st line is cut and space between lines are very small, almost no gap or overlapped.
That's because when fontSize = 15 (`fontSizeNormal`), default `lineHeight` is much bigger than 16 (`lineHeightNormal`). 18 (`lineHeightLarge`)-20(`lineHeightXLarge`) is a good fit.
So I matched `lineHeight` with composer input of which `lineHeight` = `lineHeightXLarge`.
I think we need this agreement with design team.

Reply:
There are 3 cases of multiline inputs across the app:
Workspace invite, Close account, Composer
This issue happens on Workspace invite and Close account, not happens on Composer

Reply:
For all single line inputs, `lineHeight` is ignored by this line:
https://github.com/Expensify/App/blob/3723009062e8a04a02cd261fb210e1ce578571a3/src/components/TextInput/BaseTextInput.js#L289-L291

Reply:
Hmm actually I feel like for our standard text size (15px) we should be using a line height of 20px. I'm not sure why it's called XLarge in the code, 20px = normal line height to me.

Reply:
So yeah, ideally we would be using a line height of 20px for these things but I would be cautious to see how it affects other places like text inputs, etc.

Reply:
To fix #13802, we removed `lineHeight` for all single line text inputs.
Only affected places are Workspace invite message and Close account note and Composer which have multiline text input.